web-ui/controller/git_forge.ml

```diff
@@ -68,6 +68,19 @@ module Make (View : View) = struct
         Dream.empty `Internal_Server_Error
     | Ok y -> f y
 
+  let ( >>!!= ) x f =
+    x >>= function
+    | Ok y -> f y
+    | v -> (
+        match v with
+        | Error (`Capnp ex) ->
+            Dream.log "Internal server error: %s"
+              (Fmt.to_to_string Capnp_rpc.Error.pp ex);
+            Dream.empty `Internal_Server_Error
+        | _ ->
+            (* FIXME: This is trying to match `Msg of variant *)
+            Dream.empty `Internal_Server_Error)
```

Comment:
Why not change the `>>=` Operator to support the `Error (_)` case? Otherwise, can't this be written this way to simplify the syntax?
```suggestion
  let ( >>!!= ) x f =
    x >>= function
    | Ok y -> f y
    | Error (`Capnp ex) ->
            Dream.log "Internal server error: %s"
              (Fmt.to_to_string Capnp_rpc.Error.pp ex);
            Dream.empty `Internal_Server_Error
     | Error _ ->
            (* FIXME: This is trying to match `Msg of variant *)
            Dream.empty `Internal_Server_Error)
```